src/pages/Wallet/index.jsx

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+
+import CashDisplay from '@/components/CashDisplay';
+import ItemBlock from '@/components/ItemBlock';
+import moneyData from '@/mock/money';
+import { Title, SubTitle } from '@/styles/common';
+
+import * as S from './Wallet.style';
+
+const Wallet = () => {
+  return (
+    <S.WalletWrapper>
+      <Title>MY WALLET</Title>
+      <S.BalanceWrapper>
+        <SubTitle>BALANCE</SubTitle>
+        <CashDisplay isBalance={true} />
+      </S.BalanceWrapper>
+
+      <S.MoneyWrapper>
+        <SubTitle>ADD MONEY</SubTitle>
+        <S.MoneyItemWrapper>
+          {moneyData.map(money => (
+            <ItemBlock isMoney key={money.id} unit={money.unit} count={money.count} />
+          ))}
+        </S.MoneyItemWrapper>
```

Comment:
map을 통해 리스트를 그려준다면 `ul (ol) - li` 태그가 있으면 좋을 것 같아요.
다만 여기서 `ItemBlock` 자체를 li 태그로 바꾸면 li에 종속되어버리므로 이런 방법을 사용할 수 있어요.
```suggestion
        <S.MoneyItemWrapper>
          {moneyData.map(money => (
            <li>
              <ItemBlock isMoney key={money.id} unit={money.unit} count={money.count} />
            </li>
          ))}
        </S.MoneyItemWrapper>
```